### PR TITLE
Add rusty-tags 3.5.1 release

### PR DESCRIPTION
**Pull request by janaakhterov**
###### Motivation for this change
I wanted to be able to install rusty-tags via nix instead of `cargo install`.

###### Things done
Added myself to  the list of maintainers. Added a simple nix derivation for [rusty-tags](https://github.com/dan-t/rusty-tags) v3.5.1.

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by joachifm** [comment]
Needs to be rebased. Superficially looks okay to me otherwise.

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -1153,6 +1153,11 @@
     github = "danharaj";
     name = "Dan Haraj";
   };
+  danielakhterov = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:1156]: Please put this into a separate commit with the message ``maintainers: add danielakhterov``.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -24155,4 +24155,5 @@ in
 
   bemenu = callPackage ../applications/misc/bemenu { };
 
+  rusty-tags = callPackage ../development/tools/misc/rusty-tags { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:24158]: Please don't add packages add the end but in their right categories.

File: pkgs/development/tools/misc/rusty-tags/default.nix [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  name = "rusty-tags-${version}";
```
Comment (SuperSandro2000) [inline pkgs/development/tools/misc/rusty-tags/default.nix:4]:
```suggestion
  pname = "rusty-tags";
```

File: pkgs/development/tools/misc/rusty-tags/default.nix [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  };
+
+  cargoSha256 = "1llnlr2i1xncwm55c5sg0h6fyv9991xh8iw8acyfgi5a79ylcpb6";
+
+  meta = with lib; {
+    description = "Create ctags/etags for a cargo project and all of its dependencies";
+    homepage = https://github.com/dan-t/rusty-tags;
+    license = with licenses; [ bsd3 ];
+    maintainers = with maintainers; [ danielakhterov ];
+    platforms = platforms.all;
```
Comment (SuperSandro2000) [inline pkgs/development/tools/misc/rusty-tags/default.nix:21]: This is not required here.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**